fontir/src/glyph.rs

```diff
@@ -416,8 +417,8 @@ fn flatten_glyph(context: &Context, glyph: &Glyph) -> Result<(), BadGlyph> {
     if glyph.default_instance().components.is_empty() {
         return Ok(());
     }
-    trace!(
-        "Flatten {} {:?}",
+    log::debug!(
```

Comment:
I think this should stay trace, and probably the flattening non-export components of should also be trace?

Reply:
What is your heuristic for making that determination?

my issue with trace is that if you turn it on our logging is incredibly verbose, and it's very hard to find anything. These operations don't happen that often, and I often want to see if they happen because they end up being a common source of glyf/gvar difference.